buildSrc/src/main/groovy/wala-java.gradle

```diff
@@ -27,9 +27,18 @@ repositories {
 // only use Error Prone on JDK 9+ JVMs
 final useErrorProne = JavaVersion.current().isJava9Compatible()
 
+// checks if NullAway should be used.
+def useNullAway(project) {
+	final nullawayEnrolledModules = ["com.ibm.wala.util"]
+	return nullawayEnrolledModules.contains(project.name)
```

Comment:
I don't care for using a centralized list of all enrolled modules here. Instead, this should be delegated down to the individual subprojects.

Let's create a `buildSrc/main/groovy/wala-NullAway.gradle` script. Any project that loads that script as a plugin will have the corresponding configuration changes applied to it unconditionally. For example, that script might begin with:

```groovy
plugins {
	id 'java'
}

dependencies {
	annotationProcessor 'com.uber.nullaway:nullaway:0.10.5'
}

// ...continue with any other NullAway-only project tweaks as needed...
```

For now, only `com.ibm.wala.util/build.gradle` would apply this plugin script. If we add NullAway to other subprojects later, then their corresponding `build.gradle` scripts can apply `wala-NullAway.gradle` too.